workflow-core/src/main/java/com/squareup/workflow/WorkflowPool.kt

```diff
@@ -302,15 +305,15 @@ inline fun <reified S : Any, reified E : Any, reified O : Any> WorkflowPool.regi
 
 /**
  * This is a convenience method that wraps
- * [awaitNextDelegateReaction][WorkflowPool.awaitNextDelegateReaction] in a [Deferred] so it can
+ * [WorkflowPool.awaitWorkflowUpdate] in a [Deferred] so it can
  * be selected on.
  *
- * @see WorkflowPool.awaitNextDelegateReaction
+ * @see WorkflowPool.awaitWorkflowUpdate
  */
-fun <S : Any, O : Any> WorkflowPool.nextDelegateReaction(
-  delegating: Delegating<S, *, O>
-): Deferred<Reaction<S, O>> = GlobalScope.async(Unconfined) {
-  awaitNextDelegateReaction(delegating)
+fun <S : Any, E : Any, O : Any> WorkflowPool.workflowUpdate(
+  handle: RunWorkflow<S, E, O>
+): Deferred<WorkflowHandle<S, E, O>> = GlobalScope.async(Unconfined) {
```

Comment:
Nice side-effect of returning a `Handle` instead of the `Reaction` here is the parent doesn't have the option of peeking into the child's state anymore; it's a clarifying step.